bin/dustc

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
-    dust = require('../lib/server.js'),
+    dust = require('../index.js'),
```

Comment:
I didn't want to be cute... hehe

I actually always explicitly do `.js` after @izs said in a blog post that he wished that the extension had never been made optional.